deploy/bin/import-docker-worker-secrets

```diff
@@ -0,0 +1,67 @@
+#!/bin/bash
+
+# This script reads docker-worker store in passwordstore and writes
+# up deploy.json.
+#
+# Use: ./import-docker-worker-secrets --passphrase=<your-passphrase>
+#
+# Notice you must have taskcluster passwordstore cloned and configured.
+# See ssh://gitolite3@git-internal.mozilla.org/taskcluster/secrets.git
+# for details.
+
+export PASSWORD_STORE_GPG_OPTS="$*"
+
+base_dir=/tmp
```

Comment:
setting this to `mktemp -d /dev/shm/docker-worker-XXXXX` would make me feel a lot better :)

Reply:
The problem is that this is not portable. macOS, for example, doesn't implement /dev/shm.